commit f0eed5fd326ff2ed71402e505ad6eae883926639
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-08-20T08:49:01-07:00 (7 weeks ago)
    
    TRITON-1871 vmapi migration test fail when using uuid override
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/test/lib/migration.js b/test/lib/migration.js
index 078cff0..9cfd90f 100644
--- a/test/lib/migration.js
+++ b/test/lib/migration.js
@@ -1300,7 +1300,7 @@ function TestMigrationCfg(test, cfg) {
 
         client.post({
             path: format('/vms/%s?action=migrate&migration_action=finalize',
-                targetVm.uuid)
+                sourceVm.uuid)
         }, function onMigrateFinalizeCb(err, req, res) {
             common.ifError(t, err, 'no error from migration finalize call');
             if (!err) {
